lotuspond/front/src/chain/methods.json

```diff
@@ -732,21 +732,40 @@
     "RemoveVerifiedClientDataCap"
   ],
   "fil/8/account": [
+    "Send",
+    "Constructor",
+    "PubkeyAddress",
```

Comment:
This is because we have two registered copies of the actors. @vyzo why not register the code under one CID here:

https://github.com/filecoin-project/lotus/blob/5bb50289bd3db33e1de9b1474e79ffcdae66dcf7/chain/vm/invoker.go#L167-L170

This isn't very enlightening:

https://github.com/filecoin-project/lotus/blob/5bb50289bd3db33e1de9b1474e79ffcdae66dcf7/chain/vm/invoker.go#L112

Reply:
Removing the duplicate definitions seems to have worked.